### ISSUE TITLE
Parser should accept negative numbers

### ISSUE THREAD

**Issue by nicolewhite** [nicolewhite/algebra.js#15]
The parser throws `Missing operand` when there is a negative number.

```js
var eq = algebra.parse("x^2 + 4 * x = -4");
```

```
Uncaught Error: Missing operand
    at o.construct_expression 
```

**Comment by timrach** [comment]
Is fixed in #17

**Comment by sjorsvanheuveln** [comment]
var eq = algebra.parse("5 +- 1");

This it also can't handle and throws missing operand!
